misk-crypto/src/main/kotlin/misk/crypto/CryptoModule.kt

```diff
@@ -49,7 +52,7 @@ class CryptoModule(
         KeyType.AEAD -> {
           bind<Aead>()
               .annotatedWith(Names.named(key.key_name))
-              .toProvider(AeadProvider(config.kms_uri, key))
+              .toProvider(AeadEnvelopeProvider(config.kms_uri, key))
```

Comment:
Might be worth adding documentation about this in the README file on in the `AeadKeyManager` javadoc

Reply:
👍 updated the docstring to expand a bit on operating mode.